src/data.py

```diff
@@ -241,19 +246,34 @@ def parse_static_op_hours(hours_list, eatery_id, dining_items):
     )
   return new_operating_hours
 
-def format_time(start_time, end_time, start_date):
-  start_hour, start_minute = start_time.split(':')
-  end_hour, end_minute = end_time.split(':')
+def format_time(start_time, end_time, start_date, hr24=False, overnight=False):
+  if hr24:
+    start_hour = int(start_time[0:2])
+    start_minute = start_time[2:] + 'am' if start_hour < 12 else start_time[2:] + 'pm'
+    end_hour = int(end_time[0:2])
+    end_minute = end_time[2:] + 'am' if end_hour < 12 else end_time[2:] + 'pm'
```

Comment:
there has to be a better way to parse 24 hour time into what you want, just Stack Overflow it and look for a clean solution